mingw-w64-nvc/PKGBUILD

```
@@ -0,0 +1,39 @@
+_realname=nvc
+pkgname=${MINGW_PACKAGE_PREFIX}-${_realname}
+_version="1.8_devel"
+_commit="4efbf71"
+pkgver="${_version}_${_commit}"
+pkgrel=1
+pkgdesc="VHDL compiler and simulator"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64')
+url='https://github.com/nickg/nvc'
+license=('GPL3')
+groups=("${MINGW_PACKAGE_PREFIX}-eda")
+depends=(
+  "${MINGW_PACKAGE_PREFIX}-llvm"
+  "${MINGW_PACKAGE_PREFIX}-gcc"
+)
+makedepends=(
+  "base-devel"
+  "git"
+  "autoconf"
+  "automake-wrapper"
+  "${MINGW_PACKAGE_PREFIX}-ncurses"
+  "${MINGW_PACKAGE_PREFIX}-libffi"
+  "${MINGW_PACKAGE_PREFIX}-check"
+  "${MINGW_PACKAGE_PREFIX}-pkg-config"
+)
+source=("${_realname}::git+${url}.git#commit=${_commit}")
+sha512sums=('SKIP')
+build() {
+  cd ${srcdir}/${_realname}
+  ./autogen.sh
```

```suggestion
prepare() {
  cd ${srcdir}/${_realname}
  ./autogen.sh
}
build() {
```
autogen should be part of prepare.

---

Agreed.